ddtrace/span.py

```diff
@@ -474,7 +474,7 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         # readable version of type (e.g. exceptions.ZeroDivisionError)
         exc_type_str = "%s.%s" % (exc_type.__module__, exc_type.__name__)
 
-        self.meta[ERROR_MSG] = str(exc_val)
+        self.meta[ERROR_MSG] = ensure_text(exc_val.message)
```

Comment:
Of course there must have been a gotcha between 2 and 3 😢! I think this should work for real this time! 🤞 
```suggestion
        self.meta[ERROR_MSG] = ensure_text(exc_val.message) if six.PY2 else stringify(exc_value)
```
Apologies for the mess. If that makes you feel better, this is the kind of pain we have to put up with if we want to support legacy stuff 🩸